src/app/modules/gb-analysis-module/fractalis-components/gb-fractalis-control/gb-fractalis-control.component.html

```diff
@@ -1,40 +1,56 @@
 <div class="fractalis-control-container">
+  <span *ngIf="isPreparingCache">
+    <div class="loading-text">
+      Preparing data, please wait <br/>
+      <span class="loading-blink">
```

Comment:
I just realize that this loadingblink is used everywhere, it it better to make it a standalone component, but not priority right now, just a reminder